src/source/tile_cache.js

```diff
@@ -27,9 +27,9 @@ class TileCache {
     }
 
     /**
-     * Clear the cache
+     * Clear the cache.
      *
-     * @returns {TileCache} this cache
+     * @returns {TileCache} Return `this`.
```

Comment:
For consistency with the rest of this change:
```suggestion
     * @returns {TileCache} Returns itself to allow for method chaining.
```